includes/email-templates/header.php

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html lang="en">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+  <meta name="format-detection" content="telephone=no">
+  <meta name="viewport" content="initial-scale=1.0">
+
+  <!-- So that mobile webkit will display zoomed in -->
+  <link rel="stylesheet" type="text/css" href="https://s3.amazonaws.com/web.ucf.edu/email/common-assets/stylesheet.css">
+  <title>
+    <?php the_title(); ?>
+  </title>
+
+  <?php get_template_part( 'includes/email-templates/css' ); ?>
+
+  <?php if ( is_admin_bar_showing() ) : ?>
```

Comment:
Any reason why this block (lines 17-29) can't be moved to `includes/email-templates/css`?